### ISSUE TITLE
missing modulel

### ISSUE THREAD

**Issue by Russischeir** [spyder-ide/spyder#21554]
### What steps will reproduce the problem?

hi, i can't use customtkinter, idk why, i have already installed, i saw that i have to use a miniconda envioriment to use some libraries, but i did that and just received the same error

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Program Files\Spyder\pkgs\spyder\plugins\ipythonconsole\widgets\main_widget.py", line 1176, in refresh_container
    self.infowidget.hide()
RuntimeError: wrapped C/C++ object of type FrameWebView has been deleted
```

## Versions

* Spyder version: 5.5.0  (standalone)
* Python version: 3.8.10 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.10
* Operating System: Windows-10-10.0.19045-SP0

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.1 (OK)
chardet >=2.0.0               :  5.2.0 (OK)
cloudpickle >=0.5.0           :  3.0.0 (OK)
cookiecutter >=1.6.0          :  2.4.0 (OK)
diff_match_patch >=20181111   :  20230430 (OK)
intervaltree                  :  None (OK)
IPython >=8.12.2,<8.13.0      :  8.12.3 (OK)
jedi >=0.17.2,<0.20.0         :  0.19.1 (OK)
jellyfish >=0.7               :  1.0.1 (OK)
jsonschema >=3.2.0            :  4.19.2 (OK)
keyring >=17.0.0              :  24.2.0 (OK)
nbconvert >=4.0               :  7.11.0 (OK)
numpydoc >=0.6.0              :  1.6.0 (OK)
paramiko >=2.4.0              :  3.3.1 (OK)
parso >=0.7.0,<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.6 (OK)
pygments >=2.0                :  2.16.1 (OK)
pylint >=2.5.0,<3.1           :  3.0.2 (OK)
pylint_venv >=3.0.2           :  3.0.3 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.9.0,<1.10.0         :  1.9.0 (OK)
pylsp_black >=1.2.0,<3.0.0    :  1.3.0 (OK)
qdarkstyle >=3.2.0,<3.3.0     :  3.2 (OK)
qstylizer >=0.2.2             :  0.2.2 (OK)
qtawesome >=1.2.1             :  1.2.3 (OK)
qtconsole >=5.5.0,<5.6.0      :  5.5.0 (OK)
qtpy >=2.1.0                  :  2.4.1 (OK)
rtree >=0.9.7                 :  1.1.0 (OK)
setuptools >=49.6.0           :  68.2.2 (OK)
sphinx >=0.6.6                :  7.1.2 (OK)
spyder_kernels >=2.5.0,<2.6.0 :  2.5.0 (OK)
textdistance >=4.2.0          :  4.6.0 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog                      :  3.0.0 (OK)
zmq >=22.1.0                  :  25.1.1 (OK)

# Optional:
cython >=0.21                 :  3.0.5 (OK)
matplotlib >=3.0.0            :  3.7.3 (OK)
numpy >=1.7                   :  1.24.4 (OK)
pandas >=1.1.1                :  2.0.3 (OK)
scipy >=0.17.0                :  1.10.1 (OK)
sympy >=0.7.3                 :  1.12 (OK)
```

### Environment

<details><summary>Environment</summary>

```
Package                       Version
----------------------------- ------------
alabaster                     0.7.13
arrow                         1.3.0
astroid                       3.0.1
asttokens                     2.4.1
attrs                         23.1.0
autopep8                      2.0.4
Babel                         2.13.1
backcall                      0.2.0
beautifulsoup4                4.12.2
binaryornot                   0.4.4
black                         23.10.1
bleach                        6.1.0
certifi                       2023.7.22
cffi                          1.16.0
chardet                       5.2.0
charset-normalizer            3.3.2
click                         8.1.7
cloudpickle                   3.0.0
colorama                      0.4.6
comm                          0.2.0
contourpy                     1.1.1
cookiecutter                  2.4.0
cryptography                  41.0.5
customtkinter                 5.2.1
cycler                        0.12.1
Cython                        3.0.5
darkdetect                    0.8.0
debugpy                       1.8.0
decorator                     5.1.1
defusedxml                    0.7.1
diff-match-patch              20230430
dill                          0.3.7
docstring-to-markdown         0.13
docutils                      0.20.1
et-xmlfile                    1.1.0
exceptiongroup                1.2.0
executing                     2.0.1
fastjsonschema                2.18.1
flake8                        6.1.0
fonttools                     4.44.0
idna                          3.4
imagesize                     1.4.1
importlib-metadata            6.8.0
importlib-resources           6.1.1
inflection                    0.5.1
ipykernel                     6.26.0
ipython                       8.12.3
isort                         5.12.0
jaraco.classes                3.3.0
jedi                          0.19.1
jellyfish                     1.0.1
Jinja2                        3.1.2
jsonschema                    4.19.2
jsonschema-specifications     2023.7.1
jupyter_client                8.6.0
jupyter_core                  5.5.0
jupyterlab-pygments           0.2.2
keyring                       24.2.0
kiwisolver                    1.4.5
markdown-it-py                3.0.0
MarkupSafe                    2.1.3
matplotlib                    3.7.3
matplotlib-inline             0.1.6
mccabe                        0.7.0
mdurl                         0.1.2
mistune                       3.0.2
more-itertools                10.1.0
mpmath                        1.3.0
mypy-extensions               1.0.0
nbclient                      0.9.0
nbconvert                     7.11.0
nbformat                      5.9.2
nest-asyncio                  1.5.8
numpy                         1.24.4
numpydoc                      1.6.0
openpyxl                      3.1.2
packaging                     23.2
pandas                        2.0.3
pandocfilters                 1.5.0
paramiko                      3.3.1
parso                         0.8.3
pathspec                      0.11.2
pexpect                       4.8.0
pickleshare                   0.7.5
Pillow                        10.1.0
pip                           23.3.1
pkgutil_resolve_name          1.3.10
platformdirs                  3.11.0
pluggy                        1.3.0
prompt-toolkit                3.0.39
psutil                        5.9.6
ptyprocess                    0.7.0
pure-eval                     0.2.2
pycodestyle                   2.11.1
pycparser                     2.21
pydocstyle                    6.3.0
pyflakes                      3.1.0
Pygments                      2.16.1
pylint                        3.0.2
pylint-venv                   3.0.3
pyls-spyder                   0.4.0
pyparsing                     3.1.1
PyQt5                         5.15.10
PyQt5-Qt5                     5.15.2
PyQt5-sip                     12.13.0
PyQtWebEngine                 5.15.6
PyQtWebEngine-Qt5             5.15.2
python-dateutil               2.8.2
python-lsp-black              1.3.0
python-lsp-jsonrpc            1.1.2
python-lsp-server             1.9.0
pytoolconfig                  1.2.6
pytz                          2023.3.post1
pywin32                       306
pywin32-ctypes                0.2.2
pyxdg                         0.28
PyYAML                        6.0.1
pyzmq                         25.1.1
QDarkStyle                    3.2
qstylizer                     0.2.2
QtAwesome                     1.2.3
qtconsole                     5.5.0
QtPy                          2.4.1
referencing                   0.30.2
requests                      2.31.0
rich                          13.6.0
rope                          1.11.0
rpds-py                       0.12.0
scipy                         1.10.1
setuptools                    68.2.2
six                           1.16.0
snowballstemmer               2.2.0
sortedcontainers              2.4.0
soupsieve                     2.5
Sphinx                        7.1.2
sphinxcontrib-applehelp       1.0.4
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        2.0.1
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.5
spyder                        5.5.0
spyder-kernels                2.5.0
stack-data                    0.6.3
sympy                         1.12
tabulate                      0.9.0
text-unidecode                1.3
textdistance                  4.6.0
three-merge                   0.1.1
tinycss2                      1.2.1
tomli                         2.0.1
tomlkit                       0.12.2
tornado                       6.3.3
traitlets                     5.13.0
types-python-dateutil         2.8.19.14
typing_extensions             4.8.0
tzdata                        2023.3
ujson                         5.8.0
urllib3                       2.0.7
watchdog                      3.0.0
wcwidth                       0.2.9
webencodings                  0.5.1
whatthepatch                  1.0.5
wheel                         0.41.3
yapf                          0.40.2
zipp                          3.17.0

```
</details>

**Comment by ccordoba12** [comment]
Hey @Russischeir, thanks for reporting. This is a duplicate of issue #21509 and we'll fix it in our next version (5.5.1), to be released in a couple of weeks.

**Comment by ccordoba12** [comment]
In the meantime, please ignore it.
